### PR TITLE
add TOF digitizer

### PR DESCRIPTION
**Pull request by noferini**
- Create Digitizer for TOF
- with a method to test performence of digitization

### CODE REVIEW

**Comment by noferini** [comment]
Dear all,
it is difficult for me to undestand the error. It seems that my test macro has some problems which I cannot repoduce in my local version.
Could you please give me more details on the kink of test is performed?
Thanks
Francesco

**Comment by shahor02** [comment]
@noferini : there is indeed some problem with macro compilation on macos, which build experts can clarify. But there are also these problems:

``/mnt/mesos/sandbox/sandbox/sw/SOURCES/O2/0_O2_DAQ/0/Detectors/TOF/simulation/src/Digitizer.cxx:23:18: warning: use nullptr [modernize-use-nullptr]
  HitType *hit = NULL;``

and

``error: Some files are missing the correct copyright notice on top.
error: Make sure all your source files begin with the following exact lines:
...
error: List of non-compliant files will follow.

/mnt/mesos/sandbox/sandbox/sw/SOURCES/O2/0_O2_DAQ/0/Detectors/TOF/simulation/src/Digitizer.cxx:23:18: error: use nullptr [modernize-use-nullptr]
/mnt/mesos/sandbox/sandbox/sw/SOURCES/O2/0_O2_DAQ/0//Detectors/TOF/simulation/include/TOFSimulation/Digitizer.h:1:1: error: missing or malformed copyright notice
/mnt/mesos/sandbox/sandbox/sw/SOURCES/O2/0_O2_DAQ/0//Detectors/TOF/simulation/src/Digitizer.cxx:1:1: error: missing or malformed copyright notice
``

File: Detectors/TOF/simulation/include/TOFSimulation/Digitizer.h [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+#include "TOFBase/Geo.h"
+#include "TOFSimulation/Detector.h"
+
+
+namespace o2
+{
+namespace tof
+{
+  class Digitizer : public TObject
```
Comment (sawenzel) [inline Detectors/TOF/simulation/include/TOFSimulation/Digitizer.h]: formatting looks weird. Try to run clang-format on new files.

**Comment by noferini** [comment]
I run clang on the new files (updated).

**Pull request merged**